src/components/table/Table.jsx

```diff
@@ -166,13 +166,11 @@ const Table = (props) => {
       resetScrollParams();
     });
     window.addEventListener('resize', handleWindowResize);
-    console.log('resize');
     return () => {
       window.removeEventListener('resize', handleWindowResize);
     };
-  }, []);
+  }, [selectedColumnsNames]);
```

Comment:
nope, it must be in another `useEffect`. In this case `window.addEventListener` will be called too many times

Reply:
done